<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_compare">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>华文新魏</family>
         <pointsize>15</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTipDuration">
        <number>0</number>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 255, 236, 20);
font: 15pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>开始比较</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>751</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QListWidget" name="listWidget_file1">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 255, 255, 5);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_file2">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 255, 255, 5);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>751</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QPushButton" name="pushButton_equal">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 255, 0, 20);
font: 15pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>等价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ineuqal">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 0, 255, 20);
font: 15pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>不等价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_doubt">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 0, 20);
font: 15pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>存疑</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;华文新魏&quot;;background-color: rgba(0, 255, 255, 5);</string>
    </property>
    <property name="text">
     <string>File_name1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;华文新魏&quot;;background-color: rgba(0, 255, 255, 5);</string>
    </property>
    <property name="text">
     <string>File_name2</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>640</y>
      <width>751</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QPushButton" name="pushButton_equal_2">
       <property name="text">
        <string>等价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ineuqal_2">
       <property name="text">
        <string>不等价</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_doubt_2">
       <property name="text">
        <string>存疑</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
